crates/fluvio-hub-util/src/hubaccess.rs

```diff
@@ -64,26 +65,28 @@ impl HubAccess {
         let host = &self.remote;
         let api_url = format!("{host}/{HUB_API_HUBID}");
         debug!("Sending Action token {action_token}");
-        let req = surf::put(api_url)
-            .content_type(mime::JSON)
-            .body_bytes(msg_json)
-            .header("Authorization", &action_token);
+        let req = reqwest::Client::new()
+            .put(api_url)
+            .header(CONTENT_TYPE, "application/json")
```

Comment:
I considered adding something like https://docs.rs/mime/latest/mime/ to provide a constant for `"application/json"`, I think that would be reasonable but in this case I limited myself to `reqwest`.

Reply:
I think it's fine the way it is :)